index.html

```diff
@@ -4,13 +4,80 @@
     <meta charset="UTF-8">
     <meta name="viewport" content="width=device-width, initial-scale=1.0">
     <link rel="stylesheet" href="style.css" type="text/css">
-    <title>Document</title>
+    <link href="https://fonts.googleapis.com/css2?family=Source+Sans+Pro&display=swap" rel="stylesheet">
+    <link href="https://fonts.googleapis.com/css?family=Lora" rel="stylesheet">
+    <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/4.7.0/css/font-awesome.min.css">
+
+    <title>Sensive</title>
 </head>
 <body>
+
+  <div class="main">
+    <div class="left-content">
+
+    </div>
+    <div class="right-contnet">
+
+    </div>
+
+  </div>
  
   
   <footer>
 
+      <div class="basic-content"> 
+          <div class="main-footer">
+              <div class="footer-content firstc">
+                  <h6>About</h6>
+                  <p>Lorem ipsum dolor, sit amet consectetur adipisicing elit. Ea ut dolores enim illum molestias.
+                      Quo itaque magnam voluptas aliquam,
+                      qui excepturi mollitia. Hic vitae distinctio doloremque! Modi corrupti delectus quam?
+                  </p>
+              </div>
+              <div class="footer-content">
+                  <h6>Newsletter</h6>
+                  <p>Stay update with our latest</p>
+                  <div class="container" >
+                      <input type="email" name="EMAIL" placeholder="Enter Email " class="text_input" />
+                      <button value="Save" class="btn"><i class="fa fa-angle-double-right" style="font-size: 25px; margin-top:5px"></i></button>
```

Comment:
it is not a good practice to add an inline style. you can use the class name for this icon `.fa-angle-double-right` or you can a new class name.

Reply:
I have modified it